portfolio/src/main/webapp/script.js

```diff
@@ -50,6 +50,10 @@ function addRandomFunFact() {
    const response = await fetch('/data');
    const comments = await response.json();
 
-   // display comments in comment container
-   document.getElementById('comment-container').innerText = comments.join('\n');
+   // determine what to output
+   const output = comments.length ? comments.join('\n') 
```

Comment:
nit: Kind of odd formatting for this ternary, maybe the ? section should have its own line

Reply:
Thanks! That's definitely a better choice. Fix pushed.